config/env/default.js

```diff
@@ -24,5 +24,7 @@ module.exports = {
     port: 12202,
     token: process.env.GRAYLOG_TOKEN
   },
-  historyLength: 50 
+  historyLength: 50,
+  maxMessageLength: 1000,
```

Comment:
it depends on your database capacity.
* historylength : user could spam your chat and fill up database, so you choose how long each user history is saved.
* maxMessageLength: same your db might not be happy with a huge string saved during the feedback process as each feedback is saved with the message